downstream/modules/platform/proc-upgrade-installer.adoc

```diff
@@ -0,0 +1,75 @@
+[id="proc-upgrade-installer_{context}"]
+
+= Upgrading to {PlatformNameShort} {PlatformVers}
+
+To upgrade your new instance of {PlatformNameShort} to {PlatformNameShort} {PlatformVers}, start by copying the `inventory` file from your original instance to your new instance, then running the installer. The {PlatformName} installer will detect a pre-{PlatformVers} and offer an upgraded `inventory` file for you to continue with the upgrade process:
```

Comment:
Forgive me if I'm mistaken, shouldn't this be your new instance of "Ansible Tower" to Platform, since we've just spent a bunch of time re-creating the new instance of Tower?